testcases/pi-stress.yaml

```diff
@@ -0,0 +1,12 @@
+{% extends "testcases/master/template-test.jinja2" %}
+
+{% set test_name = test_name | default("pi-stress") %}
+{% set test_path_file = 'automated/linux/pi-stress/pi-stress.yaml' %}
+{% set duration = duration | default("5m") %}
+{% set test_timeout = test_timeout | default(10) %}
+
+{% block test_target %}
+  {{ super() }}
+      parameters:
+        DURATION: '{{duration}}'
```

Comment:
Minor, but should be spaced. '{{ duration }}'

Reply:
thanks done